tools/ci/appveyor/appveyor_build.sh

```diff
@@ -21,9 +21,7 @@ BUILD_PID=0
 CACHE_DIRECTORY=/cygdrive/c/projects/cache
 
 if [[ -z $APPVEYOR_PULL_REQUEST_HEAD_COMMIT ]] ; then
-  MAKE="make -j$NUMBER_OF_PROCESSORS"
-else
-  MAKE=make
+    export MAKEFLAGS="$MAKEFLAGS -j$NUMBER_OF_PROCESSORS"
```

Comment:
No - this is potentially brittle and really surprising if one then writes `make` elsewhere expecting that to actually mean `make`! By all means rename the variable to be something clearer (e.g. `MAKE_PARALLEL` or some such) but I find exporting `MAKEFLAGS` to be in the same category as a certain opam distro's setenv on `CPP`...! 🙂